rbc/tests/test_omnisci_array.py

```diff
@@ -282,6 +282,30 @@ def array_setitem_sum(b, c):
         assert sum(f8) * 4 == s
 
 
+def test_array_constructor_return(omnisci):
+    omnisci.reset()
+
+    from rbc.omnisci_array import Array
+    from numba import types
+    from rbc.irtools import printf
+
+    @omnisci('float64[](int32)')
+    def array_return(size):
+        printf("entering array_return(%i)\n", size)
```

Comment:
The test passed but I couldn't see the `printf` messages, either locally or on the server. Imagine if there is something to do with stdout not being flushed.

Reply:
I can see it when using omniscdb 5.3 and thrift-cpp 0.13.
It is possible that in your case the messages get lost in thift-cpp < 0.13 false-positive error messages.